src/functional/program_processors/codegens/gtfn/codegen.py

```diff
@@ -64,6 +64,10 @@ class GTFNCodegen(codegen.TemplatedGenerator):
     def visit_SymRef(self, node: gtfn_ir.SymRef, **kwargs: Any) -> str:
         if node.id == "get":
             return "::gridtools::tuple_util::get"
+
+        if node.id in ["int64", "int32"]:
+            return node.id + "_t"
+
```

Comment:
I think we should do the following:
- put the builtin type to c++ name mapping into the `_builtins_mapping`
- move the logic from `visit_FunCall` that dispatches on SymRef here (needs to be adapted)
  ```        
        if isinstance(node.fun, gtfn_ir.SymRef) and node.fun.id in self._builtins_mapping:
            return self.generic_visit(node, fun_name=self._builtins_mapping[node.fun.id])
        if isinstance(node.fun, gtfn_ir.SymRef) and node.fun.id in gtfn_ir.GTFN_BUILTINS:
            qualified_fun_name = f"gtfn::{node.fun.id}"
return self.generic_visit(node, fun_name=qualified_fun_name)```